lib/foreman_ansible_core/runner/ansible_runner.rb

```diff
@@ -105,6 +107,7 @@ def write_playbook
 
       def start_ansible_runner
         command = ['ansible-runner', 'run', @root, '-p', 'playbook.yml']
+        command << '--cmdline' << '"--check"' if check_mode?
```

Comment:
What is the `--cmdline` option for? :thinking:

Reply:
--cmdline CMDLINE     command line options to pass to ansible-playbook at execution time (default=None)
This is the description from the help.
If we were using the ansible command or ansible-playbook command we could have used --check without --cmdline before but because we are using the ansible-runner this is the way to add the check mode option.